pkg/controller/configmap/config.go

```diff
@@ -288,24 +288,24 @@ func getPrefixFromCIDR(cidr string) (uint32, error) {
 }
 
 func Render(configmap *corev1.ConfigMap, ncpReplicas int32, nsxSecret *corev1.Secret,
-	lbSecret *corev1.Secret) ([]*unstructured.Unstructured, error) {
+	lbSecret *corev1.Secret) ([]*unstructured.Unstructured, error, *render.RenderData) {
```

Comment:
do you think `error` should be in the end?

Reply:
removed *render.RenderData so we don't need to care it.